src/mon/MonClient.cc

```diff
@@ -764,14 +772,19 @@ void MonClient::_renew_subs()
       sub_renew_sent = ceph_clock_now(cct);
 
     MMonSubscribe *m = new MMonSubscribe;
-    m->what = sub_have;
+    m->what = sub_new;
```

Comment:
Mmm, so it does. Are we okay not resending existing subscriptions if we're talking to an older monitor?